src/Login.js

```diff
@@ -0,0 +1,29 @@
+class Login {
+  constructor(userName, password) {
+    this.userName = userName;
+    this.password = password;
+  }
+
+  checkUserStatus(travelersData) {
+    if(this.userName.toLowerCase() === 'agent' && this.password === 'password2020') {
+      return this.userName.toLowerCase();
+    } else if(this.checkTraveler(travelersData) === true && this.password === 'password2020') {
+      return parseInt(this.userName.slice(8));
+    } else {
+      return "invalid login"
+    }
+  }
+
+  checkTraveler(travelersData) {
+    let userID = this.userName.slice(8);
+    let personData = travelersData.filter(traveler => traveler.id == userID);
+    if (personData.length === 1) {
+      return true;
+    } else {
+      return false;
+    }
+  }
```

Comment:
Nice job, I think this will work. If you plan on instantiating a Login instance each time someone logs in, that should work!

Reply:
Thanks.  I didn't think about that. That information will be useful when I continue this project.